api/src/test/java/com/ford/labs/retroquest/security/JwtAuthenticationProviderTest.java

```diff
@@ -17,23 +17,29 @@
 
 package com.ford.labs.retroquest.security;
 
-import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 import org.springframework.security.core.AuthenticationException;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.fail;
 
 class JwtAuthenticationProviderTest {
 
+    public static final String INVALID_JWT = "eyJhbGciOiJIUzUxMiJ9.eyJleHAiOjE2MjY4ODA4NzYsInN1YiI6InRlc3QiLCJpc3MiOiJSZXRyb1F1ZXN0In0.AJgA5Bhr-MZgBFyQV64BphMKJ7ScK5roBLBL7K2KSyGMl5snrhQ6yxmIwMFffC7m3aZ1kCA-y_QvaGIqru37LQ";
     private final JwtAuthenticationProvider jwtAuthenticationProvider = new JwtAuthenticationProvider();
 
     @Test
     void jwtWithInvalidFormatThrowsException() {
-        var invalidJwt = new JwtAuthentication("", false, "SOSECRET");
-        Assertions.assertThrows(
-            AuthenticationException.class,
-            () -> jwtAuthenticationProvider.authenticate(invalidJwt)
-        );
+        var invalidJwt = new JwtAuthentication(INVALID_JWT, false, "SOSECRET");
+
+        try {
+            jwtAuthenticationProvider.authenticate(invalidJwt);
+            fail();
```

Comment:
I might add an explanation argument to the fail calls here that will tell you what failed at a glance from the test output.